crates/biome_js_analyze/src/lint/nursery/use_semantic_elements.rs

```diff
@@ -59,25 +63,83 @@ impl Rule for UseSemanticElements {
             let is_not_interative =
                 AriaRoles.is_not_interactive_element(element_name, extract_attributes);
             if is_not_interative {
-                return Some(AnyJsxAttribute::from(attr));
+                let static_value = attr.as_static_value().unwrap();
+                let role_value = static_value.as_string_constant().unwrap();
+                let candidates = AriaRoles.get_corresponding_element(role_value);
+
+                if let Some(elements) = candidates {
```

Comment:
nit: you called a function that returns **an element** (singular), but you call the variable `candidates` (plural). You fix that: all plural or all singular.